config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml

```diff
@@ -28,8 +28,8 @@ limber_scrna_core_cdna_prep_input:
     - LRC PBMC Pools Input
   library_types:
     - Chromium single cell GEM-X 5p v3 GE
-limber_scrna_core_chromium_gem_x_5p_aggregation:
-  name: scRNA Core Chromium GEM-X 5p Aggregation
+limber_scrna_core_aggregation:
+  name: scRNA Core Aggregation
```

Comment:
It's odd that this change is showing up here, as this change to the 'aggregation' request type is already in the develop branch.

Any idea why this has happened?

I wonder if you pull develop into your branch whether it will fix it.

Reply:
yes, that's definitely in develop, along with the submission template further below.
why's github showing this as a change?, very odd.